wire-ios/Wire-iOS Tests/SwiftSnapshotTesting/SnapshotHelper+LightAndDarkMode.swift

```diff
@@ -0,0 +1,187 @@
+        sut.overrideUserInterfaceStyle = .light
+
+        verify(matching: sut,
+               named: name,
+               file: file,
+               testName: testName,
+               line: line)
+    }
+
+    /// Verifies the appearance of a view under dark and light themes to ensure that it appears correctly in both modes.
+    ///
+    /// - Parameters:
+    ///   - matching: The view to test..
+    ///   - name: An optional string to name the snapshot. Defaults to `nil`.
+    ///   - file: The invoking file name.
+    ///   - testName: The name of the reference image.
+    ///   - line: The invoking line number.
+
+    func verifyViewInAllColorSchemes(
+        matching: UIView,
+        file: StaticString = #file,
+        testName: String = #function,
+        line: UInt = #line
+    ) {
+        matching.overrideUserInterfaceStyle = .light
+
+        verify(matching: matching,
+               named: "LightTheme",
+               file: file,
+               testName: testName,
+               line: line)
+
+        matching.overrideUserInterfaceStyle = .dark
+
+        verify(matching: matching,
+               named: "DarkTheme",
+               file: file,
+               testName: testName,
+               line: line)
+    }
+
+    /// Verifies the appearance of a view in dark mode.
+    ///
+    /// - Parameters:
+    ///   - createSut: A closure that creates and returns the view  to be tested.
+    ///   - name: An optional string to name the snapshot. Defaults to `nil`.
+    ///   - file: The invoking file name.
+    ///   - testName: The name of the reference image.
+    ///   - line: The invoking line number.
+
+    func verifyViewInDarkScheme(
+        createSut: () -> UIView,
+        name: String? = nil,
+        file: StaticString = #file,
+        testName: String = #function,
+        line: UInt = #line
+    ) {
+        let sut = createSut()
+        sut.overrideUserInterfaceStyle = .dark
+        verify(matching: createSut(),
+               named: name,
+               file: file,
+               testName: testName,
+               line: line)
+    }
+
+    /// Verifies the appearance of a view in light mode.
+    ///
+    /// - Parameters:
+    ///   - createSut: A closure that creates and returns the view to be tested.
+    ///   - name: An optional string to name the snapshot. Defaults to `nil`.
+    ///   - file: The invoking file name.
+    ///   - testName: The name of the reference image.
+    ///   - line: The invoking line number.
+
+    func verifyViewInLightScheme(
+        createSut: () -> UIView,
+        name: String? = nil,
+        file: StaticString = #file,
+        testName: String = #function,
+        line: UInt = #line
+    ) {
+        let sut = createSut()
+        sut.overrideUserInterfaceStyle = .light
+        verify(matching: createSut(),
+               named: name,
+               file: file,
+               testName: testName,
+               line: line)
+    }
+
+}
```

Comment:
The existing verify method already allows us to specify the color scheme, this is adding another way do it. Can't we use the existing verify method?

Reply:
What do you mean by existing? As far as I understood the current ones are related to SwiftUI.

Reply:
Fixed it here: https://github.com/wireapp/wire-ios/commit/b569bd280263c125105f73db09591b3d71bdf590